docs/src/api/linear_solver_logs.md

```diff
@@ -17,6 +17,8 @@ LinSysSolverLog
 LSLogOracle
 
 LSLogFull
+
+LSLogFullMA
```

Comment:
Does this need to be called "Full"? Maybe change it in the future

Reply:
I removed the 'full' name and updated all relevant files.